src/services/api/tests/unit/test_providers_util.py

```diff
@@ -0,0 +1,10 @@
+from unittest.mock import MagicMock, patch
```

Comment:
The repo structure is a little confusing, but this technically corresponds to `src/common`, so this test isn't needed here.